.circleci/config.yml

```diff
@@ -14,7 +14,7 @@ jobs:
             - cache-pip
 
       - run: |
-          pip install numpy==1.17.4
+          # pip install numpy==1.17.4
```

Comment:
I recommend to disable these commands in a separate commit (same PR but individual test run). We do not use this work-around in the installation tests (test-install).

Reply:
Sure

Reply:
Still not sure why this change is has to be in a separate commit. Perhaps for the end PR it should be in a separate PR; that's fine. But if it's important for now, then please clarify why.

Reply:
These commands represent a work-around that is addressing an issue that is independent of our general numpy dependency constraints. That said, it appears that all of the tests pass now even without this work-around so I'm fine with completely removing this work-around (separate commit or not).